sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Storage/BlobsCheckpointStore.cs

```diff
@@ -171,7 +221,7 @@ public override async Task<IEnumerable<PartitionOwnership>> ClaimOwnershipAsync(
                     // Small workaround to retrieve the eTag.  The current storage SDK returns it enclosed in
                     // double quotes ('"ETAG_VALUE"' instead of 'ETAG_VALUE').
 
-                    Match match = s_doubleQuotesExpression.Match(ownership.ETag);
+                    var match = s_doubleQuotesExpression.Match(ownership.ETag);
```

Comment:
I wouldn't be opposed to `s_doubleQuotesExpression` going back to `DoubleQuotesExpression` or similar, as it is `static readonly` and has `const` semantics.